server/container_create_linux.go

```diff
@@ -392,12 +397,24 @@ func (s *Server) createSandboxContainer(ctx context.Context, containerID string,
 	}
 	specgen.AddAnnotation(annotations.Volumes, string(volumesJSON))
 
-	devices, err := getDevicesFromConfig(s.config)
+	cgroupMnt := rspec.Mount{
+		Destination: "/sys/fs/cgroup",
+		Type:        "cgroup",
+		Source:      "cgroup",
+		Options:     []string{"nosuid", "noexec", "nodev", "relatime", "ro"},
+	}
+	// Add cgroup mount so container process can introspect its own limits
```

Comment:
Should this part be dropped from the patch?

Reply:
probably-I wasn't sure so I left it in. I'll try dropping it